assemblies/lib/pom.xml

```diff
@@ -466,6 +466,29 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.apache.karaf.specs</groupId>
+      <artifactId>org.apache.karaf.specs.locator</artifactId>
+      <version>${karaf.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>*</groupId>
+          <artifactId>*</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>javax.annotation</groupId>
```

Comment:
Originally I tested with jakarta.annotation-api. 
Probably better to go to that instead of javax for future updates since javax.annotation-api is no longer being maintained and new updates are happening only on jakarta.annotation-api. 
https://mvnrepository.com/artifact/javax.annotation/javax.annotation-api
https://mvnrepository.com/artifact/jakarta.annotation/jakarta.annotation-api